extensions/ql-vscode/src/local-queries.ts

```diff
@@ -455,14 +455,20 @@ export class LocalQueries extends DisposableObject {
       throw new Error("Can't run query without a selected database");
     }
 
+    const additionalPacks = getOnDiskWorkspaceFolders();
+    const extensionPacks = (await this.cliServer.useExtensionPacks())
```

Comment:
If someone happens to be using the old query server, what's to prevent extension packs from being passed in?

Reply:
Nothing prevents them from being passed in, but they'll get the ~~error~~ warning from the comment below. Also, since the old query server doesn't support data extensions at all, presumably any use of extension packs would either fail (if the query actually references an extensible predicate) or would be a no-op (if the query doesn't reference an extensible predicate), right?